pkg/printers/printers.go

```diff
@@ -46,58 +45,6 @@ func GetStandardPrinter(typer runtime.ObjectTyper, encoder runtime.Encoder, deco
 			Decoders: decoders,
 		}
 
-	case "template", "go-template":
```

Comment:
How about we start right here.   What if we reverted everything and worked on this stanza here by creating the `TemplateFlags` we'd want in this one isolated spot.